include/dba.php

```diff
@@ -91,6 +91,22 @@ public function getdb() {
 		return $this->db;
 	}
 
+	/** Returns the MySQL server version string
```

Comment:
I think that once you added this small change, we finally can merge it. Hooray :-)